scripts/dataset/publishGitLab/index.js

```diff
@@ -0,0 +1,102 @@
+import fsApi from 'fs';
+import path from 'path';
+import url from 'url';
+
+import axios from 'axios';
+
+import config from 'config';
+import dotenv from 'dotenv';
+//import { Octokit } from 'octokit';
```

Comment:
Do not keep commented-out lines, please erase them.

Reply:
This is solved (along with other similar cases) and will be available when the pull request will be updated